src/flash/display/MovieClip.ts

```diff
@@ -114,6 +114,32 @@ module Shumway.AVMX.AS.flash.display {
     }
   }
 
+  export interface FrameScript {
+    (any?): any;
```

Comment:
This is simply the TypeScript way do define objects implementing FrameScript to be callable (a function).